src/Workspaces/CSharp/Portable/LanguageServices/CSharpSyntaxTreeFactoryService.cs

```diff
@@ -26,20 +26,20 @@ public CSharpSyntaxTreeFactoryService(HostLanguageServices languageServices) : b
             {
             }
 
-            public override ParseOptions GetDefaultParseOptions()
+            public override ParseOptions GetDefaultParseOptions(bool useLatestLanguageVersion)
```

Comment:
❗️ I would prefer to not change the behavior of `GetDefaultParseOptions()` (see previous comment)